schema.sql

```diff
@@ -59,3 +59,6 @@ CREATE TABLE visits (
     vet_id INTEGER REFERENCES vets(id) ON UPDATE CASCADE ON DELETE CASCADE,
     visit_date DATE NOT NULL
 );
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
- Kindly make sure that you add in your schema.sql the way to decrease the execution time for each query.  I can suggest you add indexes to solve that. follow this [link](https://dataschool.com/sql-optimization/how-indexing-works/) to learn more bout indexing :+1: 

```
e.g  
-- Add animals_id index to visits table
CREATE INDEX visits_animals_id_asc ON visits(animal_id ASC);
-- delete visits_animals_id_asc  index 
 DROP INDEX  visits_animals_id_asc;
-- Create vets_id index on visits table
CREATE INDEX visits_vets_id_asc ON visits(vet_id ASC);
-- delete  visits_vets_id_asc index 
DROP INDEX  visits_vets_id_asc;
-- CREATE email index on owners table
CREATE INDEX owners_email_asc ON owners(email ASC);
-- delete  visits_vets_id_asc index 
DROP INDEX  owners_email_asc;
```

Reply:
> * Kindly make sure that you add in your schema.sql the way to decrease the execution time for each query.  I can suggest you add indexes to solve that. follow this [link](https://dataschool.com/sql-optimization/how-indexing-works/) to learn more bout indexing 👍
> 
> ```
> e.g  
> -- Add animals_id index to visits table
> CREATE INDEX visits_animals_id_asc ON visits(animal_id ASC);
> -- delete visits_animals_id_asc  index 
>  DROP INDEX  visits_animals_id_asc;
> -- Create vets_id index on visits table
> CREATE INDEX visits_vets_id_asc ON visits(vet_id ASC);
> -- delete  visits_vets_id_asc index 
> DROP INDEX  visits_vets_id_asc;
> -- CREATE email index on owners table
> CREATE INDEX owners_email_asc ON owners(email ASC);
> -- delete  visits_vets_id_asc index 
> DROP INDEX  owners_email_asc;
> ```

Above is a quote of the previous request.